InterviewPage/js/multyAnswerQuestionCreator.js

```diff
@@ -0,0 +1,54 @@
+function createMultyAnswerQuestion() {
+    var self = {};
+
+    self.create = function (question, answers) {
+        var div = (document.createElement('div')); 
+        var $form = $('<form class="form"></form>'); 
+
+        for (var key in question.answers) {
+            $form.append(createCheckboxWithLabel(
+                question.id,
+                answers[key].id, 
+                answers[key].text,
+                question.answers[key],
+            )); 
+        }
+
+        $(div).append('<h4>' + question.text + '</h4>');  
+        $(div).append($form);
+
+        return div; 
+    };
+
+    function createCheckboxWithLabel(parentId, id, text, isChecked) {
+        var $div = $('<div></div>');
```

Comment:
This problem fixed. Moreover, html is taken out into separated file. Now, the only JavaScript responsibility is to replace key words in html templates into variables.